src/api_types.py

```diff
@@ -29,6 +29,7 @@ class DatasetMetadata(TypedDict):
 class Record(TypedDict, total=False):
     first_name: str
     last_name: str
+    is_current: bool
```

Comment:
Hmm, this field might not be present in all of the datasets (`first_name` and `last_name` were common across all of our current datasets) but seeing as it's only referenced in `augment_with_salary` I think this should be fine.

Reply:
Oh good call. Do you have any other approach that might be suitable for this? Or should we just make it an `Optional`?

Reply:
Actually shouldn't `total=false` take care of this problem?

Reply:
I think `Optional` would still imply that the field would be present everywhere, just nullable. And shoot, you're right! I was getting myself confused on what `total` was doing here. Should be fine!